content/en/real_user_monitoring/guide/understanding-the-rum-event-hierarchy.md

```diff
@@ -0,0 +1,112 @@
+
+{{< img src="real_user_monitoring/guide/understanding-rum-event-hierarchy/rum-sample-search-checkoutviewcontroller.png" alt="Diagram of the RUM event hierarchy." style="width:80%;">}}
+
+### Resources
+Resources are collected from views and include external requests from your application to a network provider, like XHR, JS loading, images, fonts, etc. Since it is collected from a view, you can query for all resources loaded on an application, or scope it down to just resources that occurred in a single view. 
+
+#### Sample search: A list of all images loaded on the /cart view, where the image size was greater than or equal to 1000 kilobytes (or 1MB)
+
+In this example, select **Resources** from the event type dropdown, then make a query for images that loaded on the cart view and were larger or equal to 1000 kilobytes.
+
+{{< img src="real_user_monitoring/guide/understanding-rum-event-hierarchy/rum-resources.png" alt="Diagram of the RUM event hierarchy." style="width:80%;">}}
+
+### Long tasks
+Long tasks are anything that blocks the UI thread for a given period of time. On mobile, for example, a long task may be a frozen frame if the screen is blocked for longer than 300 ms.
+
+#### Sample search: All frozen frame long tasks that lasted more than 500 ms
+
+In this example, select Long tasks from the event type dropdown and specify the duration.
+
+{{< img src="real_user_monitoring/guide/understanding-rum-event-hierarchy/rum-long-tasks.png" alt="Diagram of the RUM event hierarchy." style="width:80%;">}}
+
+## No data appears after writing a query
+
+{{< img src="real_user_monitoring/guide/understanding-rum-event-hierarchy/rum-no-data-appears.png" alt="Diagram of the RUM event hierarchy." style="width:80%;">}}
+
+If you aren't seeing data after writing a query, confirm that the event selector matches what you have in the search bar. In the example above, the event selector is set to search within **views**, but the search bar only contains  **action** attributes. To view action-related data, switch the view selector to actions. If you still don't see any data, check the time picker to ensure you are in a time window where data should be appearing.
+
+{{< img src="real_user_monitoring/guide/understanding-rum-event-hierarchy/rum-no-data-appears-2.png" alt="Diagram of the RUM event hierarchy." style="width:80%;">}}
+
+## How to query a event type that is nested in a different event type 
+
+When querying for specific actions, you can use the parent event type, but not one at equal or lower level. For example, actions are nested underneath views, and actions and errors are at the same level in the hierarchical chain. This means you can query for all actions and events that happened on a given page, but not for all actions that had a specific error type.
+
+#### Sample search: The top 10 actions that occurred on / (the homepage)
```

Comment:
can "/" be in code formatting? i also don't know if we need to write "the homepage"